### DISCUSSION TITLE
How can I improve the speed of SeisSol running?

### DISCUSSION THREAD

**Discussion by smallriver666** [SeisSol/SeisSol#574] [Q&A] [answered]
I hope test all the Examples in SeisSol, but any example takes too much time.
When I run SeiSsol, can I manually specify the number of mpirun, such as `./SeisSol_Release_dhsw_6_elastic parameters.par --mpirun =10` to improve the speed of SeisSol running. 
Or can the mesh be modified to increase his speed, and how should the mesh file be modified?
Thank you for your kind help.

**Comment by Thomas-Ulrich** [comment, marked as answer]
compile with order 4 and architecture skx (from your previous posts), and then run with:
```
export OMP_NUM_THREADS=20
./SeisSol_Release_dskx_4_elastic parameters.par 
```

**Reply by smallriver666** [reply]
> compile with order 4 and architecture skx (from your previous posts), and then run with:
> 
> ```
> export OMP_NUM_THREADS=20
> ./SeisSol_Release_dskx_4_elastic parameters.par 
> ```

Thank you, I will have a try tommorrow.
